src/main/command_executor.cpp

```diff
@@ -172,16 +172,14 @@ bool CommandExecutor::doCommandSingleton(Command* cmd)
       getterCommands.emplace_back(new GetProofCommand());
     }
 
-    if (options::getDumpInstantiations(d_options)
-        && ((options::getInstFormatMode(d_options)
-                 != options::InstFormatMode::SZS
-             && (res.isSat()
-                 || (res.isSatUnknown()
-                     && res.getUnknownExplanation()
-                            == api::Result::INCOMPLETE)))
-            || isResultUnsat))
+    if (options::getDumpInstantiations(d_options))
```

Comment:
I'd prefer not adding Result as an input to GetInstantiations command.

Can we just do a simple static method `static bool GetInstantiations::isEnabled(Result res)` ?

Reply:
This is a good idea, it feels way cleaner.